mlir/include/mlir/Dialect/XeGPU/IR/XeGPUAttrs.td

```diff
@@ -22,14 +22,16 @@ def XeGPU_TensorDescAttr: XeGPUAttr<"TensorDesc", "tdesc_attr"> {
   let parameters = (ins
```

Comment:
Seems like we're missing some doc for all these types and attr?

Reply:
Thanks for reminding. I added descriptions in new commits.